python/artm/batches_utils.py

```diff
@@ -70,8 +73,14 @@ def __init__(self, batches=None, collection_name=None, data_path='', data_format
                                        and if data_weight is list - automatically set to False
         :param class_ids: list of class_ids or single class_id to parse and include in batches
         :type class_ids: list of str or str
+        :param bool process_in_memory: process batches from disk or from RAM \
+                                       (only if data_format == Batches), model parameter is required
+        :param artm.ARTM model: ARTM instance that will use this vectorizer, is need only if\
+                                process_in_memory == True
         """
         self._remove_batches = False
+        self._process_in_memory = process_in_memory and data_format == 'batches' and model is not None
```

Comment:
I think we need to raise some errors if someone passes no model and process_in_memory == True, because later in *_parse_batches* we treat them as list of filenames instead of message.Batch objects.